cmd/hook/create_crd.go

```diff
@@ -0,0 +1,72 @@
+/*
+Copyright 2019 The Kubernetes Authors.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
```

Comment:
tiny nit: the boilerplate header needs to have extra newlines.

For reference: https://github.com/kubernetes/kubernetes/blob/master/hack/boilerplate/boilerplate.go.txt

Reply:
Sure.  Thanks @nikhita!